cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/AppWithMapReduceUsingRuntimeDatasets.java

```diff
@@ -84,26 +83,21 @@ public void beforeSubmit(MapReduceContext context) throws Exception {
       String outputName = runtimeArgs.get(OUTPUT_NAME);
       String outputPath = runtimeArgs.get(OUTPUT_PATH);
 
-      FileSet input;
-      FileSet output;
       if (inputName.equals(outputName)) {
```

Comment:
I was not intended to change the current code logic, but to merely use the new API instead of the now deprecated one.